cpp/tests/strings/integers_tests.cu

```diff
@@ -90,7 +230,7 @@ TEST_F(StringsConvertTest, FromInteger)
     h_expected.end(),
     thrust::make_transform_iterator(h_expected.begin(), [](auto str) { return str != nullptr; }));
 
-  CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected);
+  CUDF_TEST_EXPECT_COLUMNS_EQUAL(*results, expected, true);
```

Comment:
What does the `true` parameter here do?

Reply:
If the test fails, then all the different rows will be printed out. If you don't add that parameter, the default value is `false` and thus only one different row will be printed, which is difficult for us to fix the test.

Reply:
I'd rather keep the default here. This will keep the output for CI to a minimum and a developer could set it to `true` to temporarily debug an issue.